recipes/aioprocessing/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "aioprocessing" %}
+{% set author = "dano" %}
+{% set version = "2.0.1" %}
+{% set sha256 = "1fe91c0dca9d98e1a1280c120a7c29841f2b79fe9789239adb0d93d5200a0b60" %}
```

Comment:
Please only use name and version here and hardcode the rest. It can choke up the bot otherwise.